modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -2372,6 +2380,61 @@ validateProject <- function(projectName, configList, username, protocolName = NU
     return("")
   }
 }
+
+validateExperimentStatus <- function(experimentStatusToValidate, configList, testMode = FALSE) {
+  # validates that the supplied experiment status is in the code values
+
+  # Args:
+  #   experimentStatusToValidate: A string belonging to the experiment(type) status(kind) codetable 
+  #   configList:             Also known as racas::applicationSettings
+  #   testMode:               If true, the function bypasses Roo services and gives a database-independent answer
+  #
+  # Returns:
+  #  The experiment status code value, and the empty string if they are not
+  require('utils')
+  require('RCurl')
+  require('rjson')
+  
+  response <- NULL
+  # username <- "username"
```

Comment:
nit: Can we clean up this commented out line?

Reply:
Removed